discovery-model/src/main/java/com/ticketmaster/discovery/model/BaseModel.java

```diff
@@ -0,0 +1,22 @@
+package com.ticketmaster.discovery.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+
+public class BaseModel {
+
```

Comment:
Missing Lombok EqualsAndHashCode. Also, should have toString and subclasses should also include callsuper=true